developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -0,0 +1,3867 @@
+                    27 Aug 2012 - mcdurdin - I3438 - V9.0 - Add support for custom virtual keys
+                    27 Aug 2012 - mcdurdin - I3430 - V9.0 - Add support for if(&platform) and if(&baselayout) to compilers
+                    27 Aug 2012 - mcdurdin - I3440 - V9.0 - Tidy up set statement delimiter recognition
+                    24 Oct 2012 - mcdurdin - I3483 - V9.0 - Add support for compiling in the layout file
+                    24 Oct 2012 - mcdurdin - I3481 - V9.0 - Eliminate unsafe calls in C++
+                    24 Jan 2012 - mcdurdin - I3137 - If key part of VK rule is missing, compiler generates invalid file
+                    06 Feb 2012 - mcdurdin - I3228 - kmcmpdll sometimes tries to write temp files to Program Files
+                    03 Nov 2012 - mcdurdin - I3510 - V9.0 - Merge of I3228 - kmcmpdll sometimes tries to write temp files to Program Files
+                    03 Nov 2012 - mcdurdin - I3511 - V9.0 - Merge of I3137 - If key part of VK rule is missing, compiler generates invalid file
+                    13 Dec 2012 - mcdurdin - I3654 - V9.0 - Compiler appears to create unregistered keyboards even when registered
+                    13 Dec 2012 - mcdurdin - I3641 - V9.0 - compiler dll buffer overrun bugs
+                    13 Dec 2012 - mcdurdin - I3681 - V9.0 - KeymanWeb compiler should output formatted js when debug=1
+                    13 Dec 2012 - mcdurdin - I3686 - V9.0 - AddStore attaches property flags to wrong store structure
+                    19 Mar 2014 - mcdurdin - I4140 - V9.0 - Add keyboard version information to keyboards
+                    04 Nov 2014 - mcdurdin - I4504 - V9.0 - Consolidate the compile action into single command
+                    02 Jul 2015 - mcdurdin - I4784 - Compiler does not recognise baselayout, layer or platform at the start of a line
+                    02 Jul 2015 - mcdurdin - I4785 - baselayout(), layer() and platform() produce incorrect compiled code
+                    24 Aug 2015 - mcdurdin - I4865 - Add treat hints and warnings as errors into project
+                    24 Aug 2015 - mcdurdin - I4866 - Add warn on deprecated features to project and compile
+                    24 Aug 2015 - mcdurdin - I4867 - Add test for code after use warning to compile
+                    06 Nov 2015 - mcdurdin - I4914 - kmcmpdll does not pick an index() statement that has an offset one past the key
+                    23 Feb 2016 - mcdurdin - I4982 - Defined character constants cannot be referenced correctly in other stores
+                    25 Oct 2016 - mcdurdin - I5135 - Remove product and licensing references from Developer projects
+*/
+#include "pch.h"
+
+#include <compfile.h>
+#include <comperr.h>
+#include "../../../common/windows/cpp/include/vkeys.h"
+#include <cuchar>
+#include <versioning.h>
+#include <kmcmpdll.h>
+#include <DeprecationChecks.h>
+
+#include "virtualcharkeys.h"
+
+#include "../../../common/windows/cpp/include/crc32.h"
```

Comment:
OK, strange that it worked with wrong paths...

Reply:
yes, it's treacherous since things work on the own machine, but when someone else tries to build it doesn't work. Ran into that trap several times with changes I made...